### PR TITLE
Added InputLongAnswer component

### PR DESCRIPTION
**Pull request by meleongg**
solves #62

### CODE REVIEW

File: src/components/inputlonganswer/InputLongAnswer.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        ${(p) => getInputVariantStyles(p)}
+    `;
+
+    const ErrorMessage = styled.span`
+        ${ErrorMessageCSS}
+    `;
+
+    return (
+        <InputLongAnswer isFilled={props.isFilled} aboveCharLimit={props.aboveCharLimit} noAnswer={props.noAnswer}>
+            <Input isFilled={props.isFilled} aboveCharLimit={props.aboveCharLimit} noAnswer={props.noAnswer}>
```
Comment (ajo01) [inline src/components/inputlonganswer/InputLongAnswer.js:53]: I'm pretty sure the "Maximum of 650 characters" is supposed to be a placeholder
You can add `placeholder="Maximum of 650 characters"` to this line

File: src/components/inputlonganswer/InputLongAnswer.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        ${InputCSS}
+        ${(p) => getInputVariantStyles(p)}
+    `;
+
+    const ErrorMessage = styled.span`
+        ${ErrorMessageCSS}
+    `;
+
+    return (
+        <InputLongAnswer isFilled={props.isFilled} aboveCharLimit={props.aboveCharLimit} noAnswer={props.noAnswer}>
```
Comment (ajo01) [inline src/components/inputlonganswer/InputLongAnswer.js:52]: i'm questioning whether we need all these props. ideally, the component would be smart enough to be able to tell that it should be in an error state by checking that the text value has gone over the character limit. one way you can do this is to use react states to track the text value and the useeffect hook to check if the text value has exceeded the character limit. 
this way the only prop you'll only need is for the error state when the input is empty 

`const [text, setText] = useState("");`

`  const changeHandler = (e) => {
    setText(e.target.value);
  };`

`  useEffect(() => {
    if (text.length > 650) {
      console.log("text value over 650");
      // use a react state here to manage error for text exceeding max character limit
    }
  });`

` <Input
value={text}
        onChange={changeHandler}
</Input>`

**Review by ajo01 — CHANGES_REQUESTED** [review]

File: src/components/inputlonganswer/InputLongAnswer.js [hunk 3]
```diff
@@ -8,7 +9,27 @@ import {
 } from "./InputLongAnswerCSS";
 
 const InputLongAnswerComponent = (props) => {
-    const getInputVariantStyles = (p) => {
+    const [text, setText] = useState('');
+    const [errorText, setErrorText] = useState('');
+
+    const changeHandler = (e) => {
```
Comment (meleongg) [inline src/components/inputlonganswer/InputLongAnswer.js:15]: @ajo01 could you let me know if this state actually gets updated from the textarea onChange on your end? It wasn't working on my end when I tried

Comment (ajo01) [reply src/components/inputlonganswer/InputLongAnswer.js]: the state is getting updated. The problem is every time the text state changes the styled components are getting rerendered too so those components go out of focus. Moving the styled components above the InputLongAnswerComponent will fix this issue

**Review by ajo01 — APPROVED** [review]
looks good!

**Pull request merged**